internal/ast/kind.go

```diff
@@ -381,6 +381,7 @@ const (
 	KindJSTypeAliasDeclaration
 	KindJSExportAssignment
 	KindCommonJSExport
+	KindJSImportDeclaration
```

Comment:
Since we don’t yet have codegen in place, can you update `_packages/ast/src/syntaxKind.ts` and ``_packages/ast/src/syntaxKind.enum.ts`?